frontend/src/filter_table.tsx

```diff
@@ -72,74 +40,112 @@ const useStyles = makeStyles({
     color: 'blue',
     textDecoration: 'underline',
     cursor: 'pointer',
+  },
+  hidden: {
+    border: 0,
+    clip: 'rect(0 0 0 0)',
+    height: 1,
+    margin: -1,
+    overflow: 'hidden',
+    padding: 0,
+    position: 'absolute',
+    top: 20,
+    width: 1,
+  },
+  search: {
+    float: 'right',
   }
 });
 
+interface Column {
+  field: keyof Individual;
+  label: string;
+  sortBy?: string;
+  sortAs?: string;
```

Comment:
A bit unsure what the difference between sortBy and sortAs are here

Reply:
ah - should have commented on that. `sortBy` is to sort the column by a sub-field when the value is an object, `sortAs` is to coerce the value to a new type before sorting!